.gitignore

```diff
@@ -168,6 +168,7 @@ StyleCop.Cache
 local.settings.include
 InternalTrace.txt
 TestResult.xml
+testCaseCollection.xml
```

Comment:
One of the test runners I have installed keeps creating this file. Adding it here so it doesn't end up in the repository by mistake.